dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewManagerScopeTest.kt

```diff
@@ -514,23 +518,26 @@ internal class RumViewManagerScopeTest {
     // region AppLaunch View
 
     @Test
-    fun `𝕄 start an AppLaunch ViewScope 𝕎 handleEvent { app not displayed, event is relevant }`(
+    fun `𝕄 start an AppLaunch ViewScope 𝕎 handleEvent { app not displayed, any event }`(
         forge: Forge
     ) {
         // Given
         CoreFeature.processImportance = RunningAppProcessInfo.IMPORTANCE_FOREGROUND
         testedScope.applicationDisplayed = false
+        val appStartTimeNs = System.nanoTime() - TimeUnit.MILLISECONDS.toNanos(500)
+        whenever(mockAppStartTimeProvider.appStartTimeNs) doReturn appStartTimeNs
```

Comment:
Actually, because these tests use the time provided by the event (which is using System.nanoTime) I unfortunately can't forge the start time without changing how the mock events get their time.

I can do it but it might be some work...